tests/org.eclipse.swt.tests.win32/JUnit Tests/org/eclipse/swt/tests/win32/graphics/ImageWin32Tests.java

```diff
@@ -0,0 +1,62 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Yatta Solutions
+ *
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Yatta Solutions - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.swt.tests.win32.graphics;
+
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import org.eclipse.swt.graphics.Image;
+import org.eclipse.swt.graphics.ImageData;
+import org.eclipse.swt.internal.DPIUtil;
+import org.eclipse.swt.widgets.Display;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * Automated Tests for class org.eclipse.swt.graphics.Image
+ * for Windows specific behavior
+ *
+ * @see org.eclipse.swt.graphics.Image
+ */
+public class ImageWin32Tests {
```

Comment:
This actually only runs in Windows without any further changes (like using `Assume`)?

Reply:
FTR it does. OS-specific tests are divided in different projects: 
* `org.eclipse.swt.tests.win32`
* `org.eclipse.swt.tests.gtk`
* `org.eclipse.swt.tests.cocoa`